docs/learn/learn-async-backing.md

```diff
@@ -0,0 +1,93 @@
+---
+id: learn-async
+title: Asynchronous Backing
+sidebar_label: Asynchronous Backing
+description: A brief overview of asynchronous backing, and how it affects Polkadot's scalability.
+keywords: [parachains, slots, backing, parablock]
+slug: ../learn-async
+---
+
+Asynchronous backing is a mechanism that introduces a _pipeline_ for parachain **<->** relay chain
+communication. This pipeline will allow collators to include more data in parachain blocks while
+reducing latency (from 12 to 6 seconds for parablock validation).
+
+```mermaid
+%%{init: { 'logLevel': 'debug', 'theme': 'neutral', 'gitGraph': {'showBranches': true, 'showCommitLabel':true,'mainBranchName': 'Relay Chain'}} }%%
+      gitGraph
+        commit id:"R1"
+        branch Parachain_1 order: 2
+        commit id:"P1(C)"
+        commit id:"P1(B)"
+        branch Parachain_2 order: 3
+        checkout "Relay Chain"
+        merge Parachain_1 tag:"Parablock Backed" type:HIGHLIGHT
+        commit id:"R2"
+        checkout Parachain_2
+        commit id:"P2(C)"
+        commit id:"P2(B)"
+        checkout "Relay Chain"
+        merge Parachain_2 tag:"Parablock Backed" type:HIGHLIGHT
+        checkout Parachain_1
+        commit id:"P1(I)"
+        checkout "Relay Chain"
+        merge Parachain_1 tag:"Parablock Included" id:"R3"
+        commit id:"R4"
+        checkout Parachain_2
+        commit id:"P2(I)"
+        checkout "Relay Chain"
+        merge Parachain_2 tag:"Parablock Included" id:"R5"
+```
+
+It has three overarching goals:
+
+1. Decrease parablock validation time to **6 seconds** from **12 seconds**
+2. Increase the amount of usable blockspace by a factor of 5-10, meaning more state changes are
+   allowed per block.
+3. Allow for parachain blocks (parablocks) to be re-proposed to the network if they should be
+   included initially on the first attempt.
+
+Asynchronous backing works by providing a form of **contextual execution**, which allows for more
```

Comment:
What does "contextual execution" means exactly? How would you explain this term in your own words?

Reply:
Defined